docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -95,18 +92,19 @@ az ls-groups
 
 ## Create an alias command with arguments
 
-You can also add positional arguments to an alias command by including them as `{{ arg_name }}` in the alias name. The whitespace inside the braces is required.
+You can also add positional arguments to an alias command by including them as `{{ arg_name }}` in the alias name.
 
-```
-[alias_name {{ arg1 }} {{ arg2 }} ...]
-command = invoke_including_args
+```azurecli
+az alias create --name "alias_name {{ arg1 }} {{ arg2 }} ..." --command "invoke_including_args"
 ```
 
 The next example alias shows how to use positional arguments to get the public IP address for a VM.
 
-```
-[get-vm-ip {{ resourceGroup }} {{ vmName }}]
-command = vm list-ip-addresses --resource-group {{ resourceGroup }} --name {{ vmName }} --query [0].virtualMachine.network.publicIpAddresses[0].ipAddress
+```azurecli
+az alias create \
+    --name "get-vm-ip {{ resourceGroup }} {{ vmName }}" \
+    --command "vm list-ip-addresses --resource-group {{ resourceGroup }} --name {{ vmName }}
```

Comment:
The INI file format does __not__ support multi-line entries, so if the `--command` is stored in the passed format and does not have newlines removed, this argument must be on a single line.